.github/workflows/flake8_py_version_check.yml

```diff
@@ -0,0 +1,13 @@
+name: flake8_py_version_check
+on:
+  workflow_dispatch: {}
+  schedule:
+  # every Sunday at midnight
+  - cron: "0 0 * * 0"
+jobs:
```

Comment:
Should we explicitly install curl - Or do GitHub ubuntu workers always have curl? Ignore if so ...

Reply:
I think it's safe to assume curl is installed